pyracf/common/security_admin.py

```diff
@@ -79,11 +83,29 @@ def __init__(
             self.__replace_valid_segment_traits(replace_existing_segment_traits)
         if additional_secret_traits is not None:
             self.__add_additional_secret_traits(additional_secret_traits)
+        self.set_running_userid(run_as_userid)
+
+    # ============================================================================
+    # Run as Other User ID
+    # ============================================================================
+    def set_running_userid(self, userid: Union[str, None]) -> None:
+        if userid is None:
+            self.__running_userid = userid
```

Comment:
change to the following for additional clarity:

```python
self.__running_userid = None
```

Reply:
fair enough. This is more explicit.